spec/jobs/notifications/post_created_job_spec.rb

```diff
@@ -0,0 +1,47 @@
+require 'rails_helper'
+
+describe Notifications::PostCreatedJob do
+  subject { described_class.new(actor_id, post_id) }
+  let(:topic) { create :topic }
+  let(:post) { create :post, topic: topic }
+  let(:coach) { create :faculty }
+  let(:actor_id) { post.creator.id }
+  let(:post_id) { post.id }
+
+  describe '#perform' do
+    before do
+      create :topic_subscription, topic: topic, user: coach.user
+    end
+
+    describe 'job scheduled with an in-valid post id' do
```

Comment:
1. Use `context`, not `describe`. They're functionally the same, but one is _describing_ a thing, whereas the other is _setting up the context_ for a test.
2. _Invalid_, not _in-valid_.

```suggestion
    context 'when job is scheduled with an invalid post id' do
```